src/main/java/ch/njol/skript/structures/StructFunction.java

```diff
@@ -43,8 +55,30 @@ public class StructFunction extends Structure {
 
 	public static final Priority PRIORITY = new Priority(400);
 
+	/**
+	 * Represents a function signature pattern.
+	 * <p>
+	 * <b>Name</b>
+	 * The name may start with any Unicode alphabetic character or an underscore.
+	 * Any character following it should be any Unicode alphabetic character, an underscore, or a number.
+	 * </p>
+	 * <p>
+	 * <b>Args</b>
+	 * The arguments that can be passed to this function.
+	 * </p>
+	 * <p>
+	 * <b>Returns</b>
+	 * The type that this function returns, if any.
+	 * Acceptable return type prefixes are as follows.
+	 * <ul>
+	 *     <li>{@code ->}</li>
+	 *     <li>{@code ::}</li>
+	 *     <li>{@code returns}</li>
```

Comment:
```suggestion
	 *     <li>{@code returns}</li>
	 *     <li>{@code ->}</li>
	 *     <li>{@code ::}</li>
```
returns should be the option people first encounter, as it's the most self-explanatory (and honestly i think it should be the only option, but the symbols lobby is too powerful)